decidim-assemblies/lib/decidim/assemblies/test/factories.rb

```diff
@@ -98,4 +98,19 @@
              role: :moderator
     end
   end
+
+  factory :assembly_member, class: "Decidim::AssemblyMember" do
+    assembly { create(:assembly) }
+
+    full_name { Faker::Name.name }
+    gender { Faker::Lorem.word }
+    birthday { Faker::Date.birthday(18, 65) }
+    position { Decidim::AssemblyMember::POSITIONS.first }
+    designation_date { Faker::Date.between(1.year.ago, 1.month.ago) }
+    designation_mode { Faker::Lorem.word }
```

Comment:
Should we add the birthplace too?

Reply:
@oriolgual I can add it without problem, but I have a general doubt with factories: they should include the optional attributes?

Reply:
I'd prefer it yes, this way other developers don't have to think about it. The only exception would be the ones that change the behaviour, like what you did with `ceased`

Reply:
Thank you for the explanation, I appreciate it. Done!